microovn/api/certificates/regenerate_ca.go

```diff
@@ -34,26 +34,25 @@ func regenerateCaPut(s *state.State, r *http.Request) response.Response {
 		logger.Info("Re-issuing CA certificate and private key")
 		err = ovn.GenerateNewCACertificate(s)
 		if err != nil {
-			logger.Errorf("Failed to generate new CA certificate: %w", err)
+			logger.Errorf("Failed to generate new CA certificate: %v", err)
 			responseData.NewCa = false
 			return response.SyncResponse(false, &responseData)
-		} else {
-			responseData.NewCa = true
 		}
+		responseData.NewCa = true
 
 		// Get clients for rest of the cluster members
 		cluster, err := s.Cluster(false)
 		if err != nil {
-			logger.Errorf("Failed to get a client for every cluster member: %w", err)
+			logger.Errorf("Failed to get a client for every cluster member: %v", err)
```

Comment:
why use %v over %w

Reply:
Simply because the `logger` functions do not support `%w`, which was called out by the linter.